From 5bd00f4cbb75d256bcae9c7960ffa6efbae615d6 Mon Sep 17 00:00:00 2001
From: Pratham Deshmukh <p-deshmukh@ti.com>
Date: Thu, 12 Feb 2026 17:24:56 +0530
Subject: [PATCH 3/5] Do not re-export internal targets directories

These can be specific to a given project build and should not
be automatically inherited to dependent projects.

Upstream-Status: Pending

Signed-off-by: Pratham Deshmukh <p-deshmukh@ti.com>
---
 cmake/ONNXConfig.cmake.in | 2 --
 1 file changed, 2 deletions(-)

diff --git a/cmake/ONNXConfig.cmake.in b/cmake/ONNXConfig.cmake.in
index d588f8ae1..a61af0894 100644
--- a/cmake/ONNXConfig.cmake.in
+++ b/cmake/ONNXConfig.cmake.in
@@ -6,8 +6,6 @@
 # library version information
 set(ONNX_VERSION "@ONNX_VERSION@")
 
-list(APPEND CMAKE_PREFIX_PATH "@PROTOBUF_DIR@")
-set(Protobuf_INCLUDE_DIR "@PROTOBUF_INCLUDE_DIR@")
 find_package(Protobuf REQUIRED)
 
 # import targets
-- 
2.34.1

